### PR TITLE
Milestone 0.2.5

### PR DESCRIPTION
**Pull request by GitScrum**

### CODE REVIEW

**Comment by GitScrum** [comment]
LGTM

**Comment by GitScrum** [comment]
LGTM

File: readme.md [hunk 1]
```diff
@@ -149,14 +161,40 @@ $ posthtml -i input.html -r
 $ posthtml -i inputFolder/*.html -r
 ```
 
-## License
-MIT
+## LICENSE
```
Comment (awinogradov) [inline readme.md]: Make file LICENSE in the root of repo, please.

File: readme.md [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
+[![testen badge][testen-image]][testen-url]
+[![Trasiv Build Status][travis-image]][travis-url]
+[![AppVeyor Build Status][appveyor-img]][appveyor]
+
+[![Coveralls Status][coveralls-image]][coveralls-url]
+
+[![Dependency Status][depstat-image]][depstat-url]
+[![DevDependency Status][depstat-dev-image]][depstat-dev-url]
+
+[![XO code style][codestyle-image]][codestyle-url]
```
Comment (awinogradov) [inline readme.md]: Standard

**Comment by awinogradov** [comment]
@GitScrum you can't merge without minimum of 2 reviews. Please, make structure and linters like in https://github.com/posthtml/posthtml-plugin-boilerplate

**Pull request merged**